TeaDiary/Scripts/App/Teas/TeasDetailsPage.js

```diff
@@ -0,0 +1,62 @@
+﻿$(document).ready(() => {
+    var displayer = new TeaDisplayer({
+        Name: '#Name',
+        SecondName: '#AdditionalName',
+        Type: '#Type',
+        Notes: '#Notes'
+    });
+
+    displayer.getTeaById(currentId,
```

Comment:
`getTeaById` is a `getXxx` method. `getXxx()` returns `xxx`. Otherwise, it's not clear what happens. 
Here I see two `displayTea()` methods, one that takes the Id, another that takes the Tea object. These are two ways of doing one thing, displaying the Tea. It is clear that when you call this method, a tea is displayed. Since JS is clumsy with overloads, what if we do `loadAndDisplay(id)` and `display(object)`?